src/pqm/pqm.py

```diff
@@ -215,37 +350,49 @@ def pqm_pvalue(
         ``Ny=(1-x_frac)*(1-gauss_frac)``, and ``Ng=gauss_frac`` respectively.
         For best results, we suggest using a large number of re-tessellations,
         though this is our recommendation in any case.
+    device : str
+        Device to use for computation. Default: 'cpu'. If 'cuda' is selected,
 
     Returns
     -------
     float or list
         pvalue(s). Null hypothesis that both samples are drawn from the same
         distribution.
     """
+    # Move samples to torch tensors on the selected device
+    x_samples = torch.tensor(x_samples, device=device)
```

Comment:
for this you would use `torch.as_tensor` in case the samples are already a tensor